coriolis/tasks/osmorphing_tasks.py

```diff
@@ -72,7 +124,14 @@ def _run(self, ctxt, instance, origin, destination, task_info,
             instance_script,
             event_handler)
 
-        return {}
+        volumes_info = instance_deployment_info.get('volumes_info', [])
+        LOG.debug('Volumes info before root disk reordering: %s', volumes_info)
+        _reorder_root_disk(volumes_info, osmorphing_info.get('os_root_dev'),
```

Comment:
Having us randomly depend on `osmorphing_manager.morph_image` setting the `os_root_dev` within the mutable `osmorphing_info` is not exactly safe.

Please make `osmorphing_manager.morph_image` explicitly return the root device.
Additionally, please ensure within `osmorphing_manager.morph_image` that a copy of the `osmorphing_info` is made/mutated before being passed to the actual OSMount/OSMorphing tools.